src/forms/Form.react.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from "react";
+
+type Props = {|
+  +action: string,
+  +children?: Array<React.Node>,

```

Comment:
I believe just using `React.Node` is sufficient as its type includes array as a renderable node?

Reply:
Good call here 👍